app/models/trip.rb

```diff
@@ -0,0 +1,39 @@
+class Trip < ApplicationRecord
+  belongs_to :driver
+
+  belongs_to :passenger, optional: true
+
+  validates :rating, numericality: { only_integer: true, allow_nil: true, less_than_or_equal_to: 5, greater_than_or_equal_to: 0, message: "Rating must be a number between 0-5." }
+  # use self because we are addressing behavior not attribute. The trip object is passed into the function and then the cost is returned.
+  def self.total_cost(input)
+      total = input.cost/100
+      return "%.2f"%total
```

Comment:
I disagree that these should be `self` methods. I think it is much clearer to say something like

```ruby
@trip.total_cost
```

than

```ruby
Trip.total_cost(@trip)
```